x-pack/plugins/remote_clusters/public/sections/remote_cluster_form/remote_cluster_form.js

```diff
@@ -220,6 +256,13 @@ export class RemoteClusterFormUi extends Component {
     this.onFieldsChange({ seeds: seeds.map(({ label }) => label) });
   };
 
+  clearCurrentFields = () => {
```

Comment:
I need to update this to also clear the "Name" field, and to ignore fields which are disabled.

Reply:
Great idea! I agree we should make this common. I think the place we should make this pattern standardized is in EUI, which is something I can work on or propose later this week.